src/reconstruction/modelGeneration/test4HumanFctExt.m

```diff
@@ -800,10 +800,10 @@
     % model.c(ismember(model.rxns,'sink_glu-L(c)'))=1;
     %   FBA = optimizeCbModel(model,'max','zero');
     % TestSolution(k,1) = FBA.f;
-    TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)];
+    % TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)];
```

Comment:
Why did you comment out the failing tests?

Reply:
The failing tests were originally commented out in master - the code in master (line 752-757) is
    % model.c(ismember(model.rxns,'sink_glu-L(c)'))=1;
    %   FBA = optimizeCbModel(model,'max','zero');
    % TestSolution(k,1) = FBA.f;
    % TestSolutionName{k,1} = 'akg(c) -> glu-L(c)';
    % k = k +1;clear FBA
    %% akg[c] -> glu-L[c] % I adjusted lb since otherwise not feasible

I did some of my refactoring by search/replace, which left  TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)]; in my code uncommented. That caused the script to fail, since every test runs clear FBA (see line 770 in master, for example). I only commented the "TestedRxns" line, so it wouldn't access FBA unless FBA has just been created.